### PR TITLE
fix: android support

### PR DESCRIPTION
**Pull request by cions**

### CODE REVIEW

**Review by aapoalas — APPROVED** [review]
Looks good to me, and wonderfully harmless :)

**Comment by cions** [comment]
@aapoalas I've updated the PR. Please re-review.

File: cli/build.rs [hunk 1]
```diff
@@ -374,13 +374,20 @@ fn main() {
...
-    "Missing symbols list! Generate using tools/napi/generate_symbols_lists.js",
-  );
+  let symbols_file_name = match env::consts::OS {
+    "windows" => "generated_symbol_exports_list_windows.def".to_string(),
+    "macos" => "generated_symbol_exports_list_macos.def".to_string(),
+    "android" | "linux" => {
+      "generated_symbol_exports_list_linux.def".to_string()
+    }
+    os => format!("generated_symbol_exports_list_{}.def", os),
+  };
```
Comment (mmastrac) [inline cli/build.rs:384]:
```suggestion
  let symbols_file_name = match env::consts::OS {
    "android" => "generated_symbol_exports_list_linux.def".to_string(),
    os => format!("generated_symbol_exports_list_{}.def", os),
  };
```

**Review by mmastrac — APPROVED** [review]
LGTM, one suggestion

File: ext/net/ops.rs [hunk 2]
```diff
@@ -429,7 +429,7 @@ where
     // are different from the BSDs: it _shares_ the port rather than steal it
     // from the current listener. While useful, it's not something we can
     // emulate on other platforms so we don't enable it.
-    #[cfg(any(target_os = "windows", target_os = "linux"))]
+    #[cfg(any(target_os = "windows", target_os = "android", target_os = "linux"))]
```
Comment (mmastrac) [inline ext/net/ops.rs:432]:
```suggestion
    #[cfg(any(
      target_os = "windows",
      target_os = "android",
      target_os = "linux"
    ))]
```

**Comment by mmastrac** [comment]
@cions Thanks for the PR. Does this successfully build for Android now?

**Review by littledivy — APPROVED** [review]
LGTM!

**Comment by cions** [comment]
@mmastrac Yes, see https://github.com/cions/termux-deno

**Pull request merged**